pkgs/by-name/aq/aqualung/package.nix

```diff
@@ -0,0 +1,204 @@
+{
+  lib,
+  stdenv,
+  autoreconfHook,
+  pkg-config,
+  fetchFromGitHub,
+  nix-update-script,
+
+  # Mandatory
+  gtk3,
+  libxml2,
+
+  # Optional
+  enableLadspa ? true,
+  lrdf,
+  enableCdda ? true,
+  libcdio,
+  libcdio-paranoia,
+  enableCddb ? true,
+  libcddb,
+  enableSrc ? true,
+  libsamplerate,
+  enableLua ? true,
+  lua,
+
+  # File I/O
+  enableSndfile ? true,
+  libsndfile,
+  enableFlac ? true,
+  flac,
+  enableVorbis ? true,
+  libvorbis,
+  enableSpeex ? true,
+  liboggz,
+  speex,
+  enableMpeg ? true,
+  libmad,
+  enableLame ? true,
+  lame,
+  enableMod ? true,
+  libmodplug,
+  enableMpc ? true,
+  libmpcdec,
+  enableMac ? true,
+  monkeysAudio,
+  enableWavpack ? true,
+  wavpack,
+  enableLavc ? true,
+  ffmpeg,
+
+  # Output
+  enableSndio ? true,
+  sndio,
+  enableAlsa ? true,
+  alsa-lib,
+  enableJack ? true,
+  libjack2,
+  enablePulse ? true,
+  pulseaudio,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "aqualung";
+  version = "2.0";
+  inherit
+    enableLadspa
+    enableCdda
+    enableCddb
+    enableSrc
+    # enableIfp # FIXME: find the right dependency to support the iRiver iFP driver if it exists in nixpkgs
+    enableLua
+    enableSndfile
+    enableFlac
+    enableVorbis
+    enableSpeex
+    enableMpeg
+    enableLame
+    enableMod
+    enableMpc
+    enableMac
+    enableWavpack
+    enableLavc
+    enableSndio
+    enableAlsa
+    enableJack
+    enablePulse
+    ;
```

Comment:
These don't need to be part of the derivation itself and should be overridden via `.override` and not `.overrideAttrs`

```suggestion
```